packages/mulmocast-preprocessor/src/core/ai/context-builder.ts

```
@@ -0,0 +1,169 @@
+
+    const section = beat.meta?.section || "main";
+    if (!sections.has(section)) {
+      sections.set(section, []);
+    }
+    sections.get(section)!.push(content);
+  });
+
+  // Output by section
+  sections.forEach((contents, section) => {
+    parts.push(`## Section: ${section}`);
+    contents.forEach((c) => parts.push(c));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)